### ISSUE TITLE
Galacticraft Core causing cascading worldgen lag

### ISSUE THREAD

**Issue by FixItYondu** [micdoodle8/Galacticraft#4040]
REVIEW THE GUIDELINES - see [Contributing](https://github.com/micdoodle8/Galacticraft/blob/master/CONTRIBUTING.md) link on right side.

Minecraft version:
GalacticraftCore-1.12.2-4.0.2.280

Single player (SP), Multiplayer (MP), or SP opened to LAN (LAN)?
MP, only Forge

Any Galacticraft add-ons installed? Please list all of them: 
MicdoodleCore
Galacticraft Planets

Please upload crash report / full log to [Github Gist](https://gist.github.com/) or any other paste site:
https://gist.github.com/FixItYondu/9e13ae42987f2eeba23b6751691b701a

Please state the site that you downloaded Galacticraft from:
https://micdoodle8.com/mods/galacticraft/downloads

Basically, the warning that I pasted above is spammed in the console with different chunks each time. Is there any way to stop this spam?
